src/index.html

```diff
@@ -10,13 +10,119 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="top-navigation">
+        <a
+          href="#"
+          class="top-navigation__logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="Moyo"
+            class="top-navigation__img"
+          />
+        </a>
+
+        <ul class="top-navigation__list">
+          <li class="top-navigation__item">
+            <a
+              href="#"
+              class="top-navigation__link is-active"
```

Comment:
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```